### PR TITLE
Update workflow readability and data storage paths

### PR DESCRIPTION
**Pull request by bwmeyers**
- Update directory structure of search output products
- Improve readability and use more "best practice" approach for Nextflow scripts
- Some updates to documentation reflecting the above points

### CODE REVIEW

File: nextflow/find_candidate_position.nf [hunk 1]
```diff
@@ -343,10 +319,10 @@ process publish_best_pointing {
     publishDir params.final_dir, mode: 'copy'
 
     input:
-    file fits
+    path fits
 
     output:
-    path '*', includeInputs: true
+    path '*' includeInputs true
```
Comment (bwmeyers) [inline nextflow/find_candidate_position.nf]: @NickSwainston Can you just confirm that this formatting is correct? I had to make an educated guess to resolve a conflict.

Comment (NickSwainston) [reply nextflow/find_candidate_position.nf]: I just tried running that script and it did not give me any formatting errors so I believe it is correct

**Comment by bwmeyers** [comment]
We need at least one "Approved" review, and it doesn't make sense for it to be me :sweat_smile:

**Review by NickSwainston — APPROVED** [review]

**Comment by bwmeyers** [comment]
For reference, I'm waiting on merging this until we also make the required changes in `vcstools` (which I think are mostly done).

**Comment by bwmeyers** [comment]
Now that vcstools changes are passing and approved, we can merge this and deploy.

**Pull request merged**